<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Question2</class>
 <widget class="QDialog" name="Question2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Question2</string>
  </property>
  <widget class="QCalendarWidget" name="CalendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>321</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a date to determine the days remaining in this year:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalculateBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="DaysRemainingTxt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Days remaining in this year...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CalendarWidget</sender>
   <signal>selectionChanged()</signal>
   <receiver>CalculateBtn</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
